lavanet/lava
# PRT-514 quorum select first provider

## Description
This PR implement first of many strategies we are going to have for sending relays. The strategy which is implemented in this PR Is LATENCY. 

## How it works:
When a user sends a request to the consumer. The consumer will pick a subset of providers (for now hardcoded to 2) and it will send a relay to all selected providers. Whoever returns the first non-error response it will be served to the client. If all providers return an error the last response is returned to the consumer

## Manual tests:
1. Start local network with 3 providers and set optimiser to return 2 providers
2. Execute queries multiple times, and for each request 2 providers needs to get return
3. Stop 1 provider and execute multiple requests, both active needs to get payment for each
4. Stop 1 more, now we only have 1 active which is less than optimiser is returning, that one should still serve relays and get payment
